src/lib/packet.h

```diff
@@ -355,6 +355,7 @@ typedef enum {
     PGP_PKA_RESERVED_DH = 21,             /* Reserved for Diffie-Hellman
                                            * (X9.42, as defined for
                                            * IETF-S/MIME) */
+    PGP_PKA_EDDSA = 22,                   /* EdDSA */
```

Comment:
@catap indeed. The point is the IETF OpenPGP group is not going to publish https://tools.ietf.org/html/draft-koch-eddsa-for-openpgp-04 but this will be incorporated directly into https://tools.ietf.org/html/draft-ietf-openpgp-rfc4880bis-01, so it's better if we refer to the later one. 👍